orchestrator/graphql/schemas/subscription.py

```diff
@@ -166,3 +93,16 @@ async def depends_on_subscriptions(
             GraphqlFilter(field="subscriptionIds", value=",".join(subscription_ids))
         ]
         return await resolve_subscriptions(info, filter_by_with_related_subscriptions, sort_by, first, after)
+
+
+@strawberry.experimental.pydantic.type(model=SubscriptionModel, all_fields=True)
+class SubscriptionPydantic(SubscriptionInterface):
+    pass
+
+
+@strawberry.federation.type(description="Virtual base class for subscriptions", keys=["subscriptionId"])
+class Subscription(SubscriptionPydantic):
```

Comment:
Should this be `class SubscriptionGraphql`? For consistency with the other types